lib/src/app_bar.dart

```diff
@@ -144,18 +189,27 @@ class BackdropAppBar extends StatelessWidget implements PreferredSizeWidget {
       flexibleSpace: flexibleSpace,
       bottom: bottom,
       elevation: elevation,
+      shadowColor: shadowColor,
       shape: shape,
       backgroundColor: backgroundColor,
+      foregroundColor: foregroundColor,
+      // ignore: deprecated_member_use, deprecated_member_use_from_same_package
       brightness: brightness,
```

Comment:
`brightness` and `textTheme` are deprecated but not removed by flutter. Will do the same - will remove when flutter removes it.

Reply:
@WieFel

Reply:
LGTM